### ISSUE TITLE
can u make -it work on odoo 10

### ISSUE THREAD

**Issue by ghost** [odoospace/radius#2]
hi , 
can u make module to work in odoo 10

**Comment by lasarux** [comment]
Hi, this module is being developed for odoo 8 at first place. After this it'll updated to newer odoos. I'm interested in your feedback in order to get a better radius module for odoo.

**Comment by ghost** [comment]
Yes we are a small isp in romania and we want to migrate to odoo and for this we will like to have a radius module to be able to manage the pppoe custommers from radius

**Comment by ghost** [comment]
any news regarding to move code to odoo 10

**Comment by ghost** [comment]
any news

**Comment by lasarux** [comment]
Next week will begin the development for odoo 10...

**Comment by ghost** [comment]
ok thank you. i will wait

**Comment by ghost** [comment]
[radius_db.zip](https://github.com/odoospace/radius/files/1654768/radius_db.zip)

i have attached a database of freeradius 3

**Comment by ghost** [comment]
any news

**Comment by koz01** [comment]
hello, Can you upgrade this module for Odoo13?
Please contact me via Skype.
live:29dda3336b07b4fb
